src/rule.js

```diff
@@ -10,9 +10,11 @@
         email: /^([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/,
         number: /^\d+$/,
         minlength: function(value, scope, element, attrs, param) {
+          value += '';
```

Comment:
> So Ill remove the string conversion, update the related tests and squash the commits to single one.

> 

Done

> more in detail which lines should be changed? (If not already discussed above) Thanks :)
Sure.

If there's anything else left to be done, let me know please :) Thanks.